scripts/windows/test.ps1

```diff
@@ -0,0 +1,15 @@
+#Requires -Version 5.0
+$ErrorActionPreference = "Stop"
+
+Invoke-Expression -Command "$PSScriptRoot\version.ps1"
+
+$DIR_PATH = Split-Path -Parent $MyInvocation.MyCommand.Definition
+$SRC_PATH = (Resolve-Path "$DIR_PATH\..\..").Path
+cd $SRC_PATH
+
+$env:GOARCH=$env:ARCH
+$env:GOOS='windows'
+go test -v -cover -tags 'test' ./pkg/agent/...
+if (-not $?) {
+    throw "go test failed"
+}
```

Comment:
we aren't running any of the integation tests. why?

Reply:
Windows cluster is still a mixed cluster: Linux control plane + Windows worker. I don't know how to set up a Linux control plane on Windows to do integration testings. Is it possible for now?